examples/flink-consumer-source/src/main/scala/org/apache/flink/batch/connectors/pulsar/example/README.md

```diff
@@ -279,5 +279,67 @@ Please find sample output for above application as follows:
 
 ### Complete Example
 
-You can find a complete example [here](https://github.com/apache/incubator-pulsar/tree/master/pulsar-flink/src/test/scala/org/apache/flink/batch/connectors/pulsar/example/FlinkPulsarBatchJsonSinkScalaExample.scala).
+You can find a complete example [here](https://github.com/apache/pulsar/tree/master/examples/flink-consumer-source/src/main/scala/org/apache/flink/batch/connectors/pulsar/example/FlinkPulsarBatchJsonSinkScalaExample.scala).
 In this example, Flink DataSet is processed and written to Pulsar in Json format.
+
+
+# PulsarJsonOutputFormat
+### Usage
+
+Please find Scala sample usage of `PulsarJsonOutputFormat` as follows:
+
+```scala
+      val nasaMissions = List(
+          NasaMission.newBuilder.setId(1).setName("Mercury program").setStartYear(1959).setEndYear(1963).build,
+          NasaMission.newBuilder.setId(2).setName("Apollo program").setStartYear(1961).setEndYear(1972).build,
+          NasaMission.newBuilder.setId(3).setName("Gemini program").setStartYear(1963).setEndYear(1966).build,
+          NasaMission.newBuilder.setId(4).setName("Skylab").setStartYear(1973).setEndYear(1974).build,
+          NasaMission.newBuilder.setId(5).setName("Apollo–Soyuz Test Project").setStartYear(1975).setEndYear(1975).build)
+      
+        def main(args: Array[String]): Unit = {
+      
+          // set up the execution environment
+          val env = ExecutionEnvironment.getExecutionEnvironment
+      
+          // create PulsarCsvOutputFormat instance
+          val pulsarAvroOutputFormat =
+            new PulsarAvroOutputFormat[NasaMission](SERVICE_URL, TOPIC_NAME)
+      
+          // create DataSet
+          val textDS = env.fromCollection(nasaMissions)
+      
+          // map nasa mission names to upper-case
+          textDS.map(nasaMission => new NasaMission(
+            nasaMission.getId,
+            nasaMission.getName,
+            nasaMission.getStartYear,
+            nasaMission.getEndYear))
+      
+            // filter missions which started after 1970
+            .filter(_.getStartYear > 1970)
+      
+            // write batch data to Pulsar as Avro
+            .output(pulsarAvroOutputFormat)
+      
+          // set parallelism to write Pulsar in parallel (optional)
+          env.setParallelism(2)
+      
+          // execute program
+          env.execute("Flink - Pulsar Batch Avro")
+        }
+```
+
+**Note:** Property definitions of the model should cover `@BeanProperty` to be visible.
```

Comment:
`@BeanProperty` is required for **Json** format to be visible properties for serialization. So, this note needs to be removed for Avro.

Reply:
ok